### PR TITLE
{CI} Fix top N slowest Test

### PR DESCRIPTION
**Pull request by wangzelin007**
**Description**

This PR will speed up our test.

- Our list command is not pagination, it will get all the data.
   That is why some recording files are very large.
   And in test_show_built_in_policy, we choose to check the first resource instead of check all the resource to speed up.  

- We only mock time.sleep in azure-sdk-tools.
   That's why if someone use sleep(xxx), it will not work.
   https://github.com/Azure/azure-cli/blob/30216d0268f3491b8af2aa060d90a9e1f94f9b3c/src/azure-cli-testsdk/azure/cli/testsdk/scenario_tests/patches.py#L13

| Profile | Test Case | Before | After | Total saving time |
|--------|------------|---------|-------|--------------------|
| Latest | test_show_built_in_policy | 536.57s | 2.94s |  |
| 2019-03-01 | test_show_built_in_policy | 210.69s | 3.01s |  |
| Latest | test_sql_db_restore | 305.01s | 8.70s ||
| Latest | test_sql_db_restore_deleted | 184.63s | 7.39s ||
| Latest | test_postgrsql_general_purpose_replica_mgmt | 121.76s | 4.11s ||
| Latest | test_sql_tde | 10.54s | 3.03s ||
| Latest | test_sql_ledger | 7.62s | 3.00s ||
| Total |||| 1,344.64s |

image

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
CI

File: src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py [hunk 1]
```diff
@@ -1326,7 +1326,7 @@ def _create_db_wait_for_first_backup(test, resource_group, server, database_name
     # Wait until earliestRestoreDate is in the past. When run live, this will take at least
     # 10 minutes. Unforunately there's no way to speed this up
     while db['earliestRestoreDate'] is None:
-        sleep(60)
+        time.sleep(60)
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py:1329]: `from time import sleep` can now be removed.

Comment (wangzelin007) [reply src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py:1329]: We need modify these codes before remove `from time import sleep`.
I think add `if self.is_live` condition is also acceptable.
https://github.com/Azure/azure-cli/blob/92e45246c6b0e635bb7c175f90e8ceeb6702972b/src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py#L1125-L1126
https://github.com/Azure/azure-cli/blob/92e45246c6b0e635bb7c175f90e8ceeb6702972b/src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py#L1454-L1456
https://github.com/Azure/azure-cli/blob/92e45246c6b0e635bb7c175f90e8ceeb6702972b/src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py#L2743-L2744
https://github.com/Azure/azure-cli/blob/92e45246c6b0e635bb7c175f90e8ceeb6702972b/src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py#L2764-L2765
https://github.com/Azure/azure-cli/blob/92e45246c6b0e635bb7c175f90e8ceeb6702972b/src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py#L2777-L2778
https://github.com/Azure/azure-cli/blob/92e45246c6b0e635bb7c175f90e8ceeb6702972b/src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py#L4481-L4482
https://github.com/Azure/azure-cli/blob/92e45246c6b0e635bb7c175f90e8ceeb6702972b/src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py#L4594-L4595
https://github.com/Azure/azure-cli/blob/92e45246c6b0e635bb7c175f90e8ceeb6702972b/src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py#L4607-L4608

**Review by evelyn-ys — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py (hunk 1 above)
Comment (jiasli) [reply src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py:1329]: Let's only remove `from time import sleep` when it is not used.

**Review by zhoxing-ms — APPROVED** [review]

**Review by calvinhzy — APPROVED** [review]

**Pull request merged**